### ISSUE TITLE
SpreadsheetConvertersConverterProvider should support "General"

### ISSUE THREAD

**Issue by mP1** [mP1/walkingkooka-spreadsheet#4416]
Not sure how to "give" the parsers and formatters required.

**Comment by mP1** [comment]
created by

```
return SpreadsheetConverters.general(
                spreadsheetFormatterProvider.spreadsheetFormatterOrFail(dateFormat),
                spreadsheetParserProvider.spreadsheetParserOrFail(dateParser),
                spreadsheetFormatterProvider.spreadsheetFormatterOrFail(dateTimeFormat),
                spreadsheetParserProvider.spreadsheetParserOrFail(dateTimeParser),
                spreadsheetFormatterProvider.spreadsheetFormatterOrFail(numberFormat),
                spreadsheetParserProvider.spreadsheetParserOrFail(numberParser),
                spreadsheetFormatterProvider.spreadsheetFormatterOrFail(textFormat),
                spreadsheetFormatterProvider.spreadsheetFormatterOrFail(timeFormat),
                spreadsheetParserProvider.spreadsheetParserOrFail(timeParser)
        );
```

**Comment by mP1** [comment]
Closed by #4438
